redisinsight/ui/src/pages/autodiscover-cloud/redis-cloud-subscriptions/components/CancelButton/CancelButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { SecondaryButton, DestructiveButton } from 'uiSrc/components/base/forms/buttons'
+import { Text } from 'uiSrc/components/base/text'
+import { RiPopover } from 'uiSrc/components/base'
+
+import styles from '../../styles.module.scss'
+import { type CancelButtonProps } from './CancelButton.types'
+
+export const CancelButton = ({
+  isPopoverOpen,
+  onClose,
+  onShowPopover,
+  onClosePopover,
+}: CancelButtonProps) => (
+  <RiPopover
+    anchorPosition="upCenter"
+    isOpen={isPopoverOpen}
+    closePopover={onClosePopover}
+    panelClassName={styles.panelCancelBtn}
+    panelPaddingSize="l"
+    button={
+      <SecondaryButton
+        onClick={onShowPopover}
+        className="btn-cancel"
+        data-testid="btn-cancel"
+      >
+        Cancel
+      </SecondaryButton>
+    }
+  >
+    <Text size="m">
+      Your changes have not been saved.&#10;&#13; Do you want to proceed to
```

Comment:
nit: Can't we use a `<br />` or two separate `Text` components.

PS: Still, it was a nice feeling when I saw 10/13 and knew what it is all about 😀